resources/lib/PayoneApi/Request/GenericPayment/GenericPaymentRequestFactory.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace PayoneApi\Request\GenericPayment;
+
+use PayoneApi\Lib\Version;
+use PayoneApi\Request\Parts\RedirectUrls;
+use PayoneApi\Request\Parts\SystemInfo;
+use PayoneApi\Request\Parts\Config;
+use PayoneApi\Request\RequestFactoryContract;
+
+class GenericPaymentRequestFactory implements RequestFactoryContract
+{
+
+    /**
+     * @param string $paymentMethod
+     * @param array $data
+     * @param string|bool $referenceId Reference to previous request
+     *
+     * @return mixed
+     */
+    public static function create($paymentMethod, $data, $referenceId = null)
```

Comment:
Typen fehlen

Reply:
Gibt das Interface nicht her, deswegen stehen sie nur in den Annotations.